src/Util/HtmlUtils.php

```diff
@@ -36,13 +36,17 @@ public function inlineCss($html, $css)
             $css = implode(PHP_EOL, $css);
         }
 
-        $this->cssToInlineStyles->setHTML($html);
-        $this->cssToInlineStyles->setCSS($css);
+        if (\method_exists($this->cssToInlineStyles, 'setHTML')) {
```

Comment:
Why this backslash?

Reply:
https://secure.php.net/manual/en/language.namespaces.faq.php#language.namespaces.faq.shortname2

It optionnal, until `method_exists` isn’t defined in the current namespace, there is no difference. But, I think, it’s a good habit.

Reply:
Since the bundle never used the backslash for php functions I think we can strip it for now. Good to know by the way.